typespec-extension/src/code-model-builder.ts

```diff
@@ -1570,53 +1574,17 @@ export class CodeModelBuilder {
       } else {
         // schema (usually JSON)
         let schema: Schema | undefined = undefined;
-        const verb = op.requests?.[0].protocol?.http?.method;
-        if (
-          // LRO, candidateResponseSchema is Model/ObjectSchema
-          candidateResponseSchema &&
-          candidateResponseSchema instanceof ObjectSchema &&
-          // bodyType is templated Model
-          bodyType.kind === "Model" &&
-          bodyType.templateMapper &&
-          bodyType.templateMapper.args &&
-          bodyType.templateMapper.args.length > 0
-        ) {
-          if (verb === "post") {
-            // for LRO ResourceAction, the standard does not require a final type, hence it can be the same as intermediate type
-            // https://github.com/microsoft/api-guidelines/blob/vNext/azure/ConsiderationsForServiceDesign.md#long-running-action-operations
-
-            // check if we can use candidateResponseSchema as response schema (instead of the templated Model), for LRO ResourceAction
-            if (candidateResponseSchema.properties?.length === bodyType.properties.size) {
-              let match = true;
-              for (const prop of Array.from(bodyType.properties.values())) {
-                const name = this.getName(prop);
-                if (!candidateResponseSchema.properties?.find((it) => it.language.default.name === name)) {
-                  match = false;
-                  break;
-                }
-              }
-              if (match) {
-                schema = candidateResponseSchema;
-                const sdkType = getClientType(this.sdkContext, bodyType);
-                const modelName = sdkType.kind === "model" ? sdkType.name : undefined;
-                this.trace(
-                  `Replace TypeSpec model '${modelName}' with '${candidateResponseSchema.language.default.name}'`,
-                );
-              }
-            }
-          } else if (verb === "delete") {
```

Comment:
remove this would cause some minor Javadoc change on response, e.g.
https://github.com/Azure/autorest.java/pull/2801/files#diff-d5c9a91efac5de9c40e886638c59257e53f04b3fd9670ef7a8464daeb1866351